zzz_generated/controller-clusters/zap-generated/CHIPClusters.cpp

```diff
@@ -138,7 +138,7 @@ CHIP_ERROR AccountLoginCluster::Login(Callback::Cancelable * onSuccessCallback,
     app::CommandPathParams cmdParams = { mEndpoint, /* group id */ 0, mClusterId, AccountLogin::Commands::Ids::Login,
                                          (app::CommandPathFlags::kEndpointIdValid) };
 
-    SuccessOrExit(err = app::InteractionModelEngine::GetInstance()->NewCommandSender(&sender));
+    SuccessOrExit(err = mDevice->NewCommandSender(&sender));
```

Comment:
Like I mentioned earlier, just call `new` or Platform::New directly here. Let's reduce the # of layers here and simplify.